tests/phpunit/CRM/CiviAwards/Service/ApplicantManagementCustomGroupDisplayFormatterTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+use CRM_CiviAwards_Helper_CustomGroupPostProcess as ApplicantManagementPostProcessHelper;
+use CRM_CiviAwards_Service_ApplicantManagementCustomGroupDisplayFormatter as ApplicantManagementCustomGroupDisplayFormatter;
+use CRM_CiviAwards_Setup_ProcessAwardsCategoryForCustomGroupSupport as CaseCategoryForCustomGroupSupport;
+
+/**
+ * Runs tests for ApplicantManagementCustomGroupDisplayFormatterTest.
+ *
+ * @group headless
+ */
+class CRM_CiviAwards_Service_ApplicantManagementCustomGroupDisplayFormatterTest extends BaseHeadlessTest {
+
+  /**
+   * Test process display function.
+   */
+  public function testProcessDisplay() {
+    $caseCategories = [3 => 'Awards'];
+    $expectedDisplay = CaseCategoryForCustomGroupSupport::AWARDS_CATEGORY_CG_LABEL;
+    $cgExtends = ['Awards' => $expectedDisplay];
+    $subTypes = [3 => [1 => 1, 2 => 2]];
+    $awardManagementHelper = $this->getApplicantManagementHelperMock($caseCategories, $cgExtends, $subTypes);
+    $row = ['id' => 3, 'extends_entity_column_id' => 3];
+    $displayFormatter = $this->getMockBuilder(ApplicantManagementCustomGroupDisplayFormatter::class)
```

Comment:
Can we move this to a new method? `getDsiplayFormatter`

Reply:
fixed